internal/util/prompt.go

```diff
@@ -250,23 +250,25 @@ func addAnotherContractToDeploymentPrompt() bool {
 // ShowContractDiffPrompt shows a diff between the new contract and the existing contract
 // and asks the user if they wish to continue with the deployment
 // returns true if the user wishes to continue with the deployment and false otherwise
-func ShowContractDiffPrompt(newContract []byte, existingContract []byte) bool {
-	dmp := diffmatchpatch.New()
-	diffs := dmp.DiffMain(string(newContract), string(existingContract), false)
-	diffString := dmp.DiffPrettyText(diffs)
-	fmt.Println(diffString)
+func ShowContractDiffPrompt(logger *output.Logger) func([]byte, []byte) bool {
```

Comment:
`output.Logger` is an interface so we can drop the pointer, This will result in a simpler syntax.